libraries/AP_Networking/config/lwipopts.h

```diff
@@ -70,21 +75,27 @@ extern "C"
 #define LWIP_NETCONN               (NO_SYS==0)
 #define LWIP_NETIF_API             (NO_SYS==0)
 
+#ifndef LWIP_IGMP
 #define LWIP_IGMP                  LWIP_IPV4
+#endif
 #define LWIP_ICMP                  LWIP_IPV4
 
 #define LWIP_SNMP                  LWIP_UDP
 #ifdef LWIP_HAVE_MBEDTLS
 #define LWIP_SNMP_V3               (LWIP_SNMP)
 #endif
 
-#define LWIP_DNS                   LWIP_UDP
-#define LWIP_MDNS_RESPONDER        LWIP_UDP
+#define LWIP_DNS                   0
```

Comment:
Why disable DNS for all platforms? I know we're not using it right now so it shouldn't consume much memory.. but I plan to use SNTP and likely others in the future. Bootloader-wise, UDP isn't enabled so this is already disabled.

Reply:
because we have no use of DNS at the moment. When we have exposed APIs for DNS then we can enable this, but for now its just a waste of flash